test/plugin/scenarios/webflux-scenario/webflux-projectA-scenario/src/main/java/org/apache/skywalking/apm/testcase/sc/webflux/projectA/controller/TestController.java

```diff
@@ -43,6 +43,7 @@ public String testcase() throws IOException {
         visit("http://" + hostBAddress + "/testcase/route/error");
         visit("http://" + hostBAddress + "/notFound");
         visit("http://" + hostBAddress + "/testcase/annotation/mono/hello");
+        visit("http://" + hostBAddress + "/testcase/webclient/test1");
```

Comment:
If we lost the context before, this is the place broke. But it doesn't seem the case. Webflux plugin still guarantee the spring mvc has correct context, isn't it?

Reply:
en It‘s ’weird. Are you sure the other trace segment is right？(exclude the test I added)

Reply:
I am not. But webflux server plugin works in Gateway, nd springmvc in the real world. Others, I am not sure.

Reply:
I mean the trace segment in expectedData.